src/app/store/effects/mail.effects.ts

```diff
@@ -166,6 +165,15 @@ export class MailEffects {
         );
     });
 
+  @Effect()
```

Comment:
why did you write a new action `SetMailDetail`, because you could have directly dispatched `GetMailDetailSuccess`,
It doesn't make sense to make a new action `SetMailDetail` which will eventually dispatch `GetMailDetailSuccess`.